docs/model_monitoring/monitoring.md

```diff
@@ -2,11 +2,3 @@
 
 > this section is under constructions !
 
-**In this section**
-
```

Comment:
if u replaced this with `index.md` so why keep this file?

Reply:
I'm fine removing it, I think we should revise monitoring later, but we have a lot to fix anyhow prior to that.

Reply:
ok, now i see, u referred to the old `index.md`, I don't like the content in it (igz and drift specific), if we we drop `monitoring.md` we should use its content instead of whats in in `index.md` (and we will add more later). and YES I do want a reference to the tutorial in it since its the best example/doc we have